tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_DampedHarmonicGaugeQuantities.cpp

```diff
@@ -0,0 +1,1326 @@
+      static_cast<typename db::item_type<
+          GeneralizedHarmonic::Tags::SpacetimeDerivGaugeH<SpatialDim,
+                                                          Frame>> (*)(
+          const typename db::item_type<
+              GeneralizedHarmonic::Tags::InitialGaugeH<SpatialDim, Frame>>&,
+          const typename db::item_type<
+              GeneralizedHarmonic::Tags::SpacetimeDerivInitialGaugeH<SpatialDim,
+                                                                     Frame>>&,
+          const Scalar<DataVector>&,
+          const tnsr::I<DataVector, SpatialDim, Frame>&,
+          const tnsr::a<DataVector, SpatialDim, Frame>&,
+          const Scalar<DataVector>&,
+          const tnsr::II<DataVector, SpatialDim, Frame>&,
+          const tnsr::aa<DataVector, SpatialDim, Frame>&,
+          const tnsr::aa<DataVector, SpatialDim, Frame>&,
+          const tnsr::iaa<DataVector, SpatialDim, Frame>&, const double,
+          const double, const double,
+          const tnsr::I<DataVector, SpatialDim, Frame>&, const double)>(
+          &::GeneralizedHarmonic::SpacetimeDerivDampedHarmonicHCompute<
+              SpatialDim, Frame>::function),
+      "TestFunctions", "spacetime_deriv_damped_harmonic_gauge_source_function",
+      {{{0.1, 10.}}}, used_for_size, 1.e-11);
+}
+
+// Test term 1/4 of spacetime deriv of gauge source function for Kerr-Schild
+template <typename Solution>
+void test_deriv_damped_harmonic_h_function_term_1_of_4_analytic(
+    const Solution& solution, const size_t grid_size_each_dimension,
+    const std::array<double, 3>& lower_bound,
+    const std::array<double, 3>& upper_bound) noexcept {
+  // Initialize random number generation
+  std::random_device rd;
+  const auto seed = rd();
+  INFO("seed" << seed);
+  std::mt19937 generator(rd());
+  std::uniform_real_distribution<> rdist(0.1, 1.);
+  std::uniform_real_distribution<> pdist(0.01, 1.);
+
+  // Setup grid
+  const size_t SpatialDim = 3;
+  Mesh<SpatialDim> mesh{grid_size_each_dimension, Spectral::Basis::Legendre,
+                        Spectral::Quadrature::GaussLobatto};
+
+  using Affine = CoordinateMaps::Affine;
+  using Affine3D = CoordinateMaps::ProductOf3Maps<Affine, Affine, Affine>;
+  const auto coord_map =
+      make_coordinate_map<Frame::Logical, Frame::Inertial>(Affine3D{
+          Affine{-1., 1., lower_bound[0], upper_bound[0]},
+          Affine{-1., 1., lower_bound[1], upper_bound[1]},
+          Affine{-1., 1., lower_bound[2], upper_bound[2]},
+      });
+
+  // Setup coordinates
+  const auto x_logical = logical_coordinates(mesh);
+  const auto x = coord_map(x_logical);
+  // Arbitrary time for time-independent solution.
+  const double t = pdist(generator) * 0.4;
+
+  // Evaluate analytic solution
+  const auto vars =
+      solution.variables(x, t, typename Solution::template tags<DataVector>{});
+  const auto& lapse = get<gr::Tags::Lapse<>>(vars);
+  const auto& dt_lapse = get<Tags::dt<gr::Tags::Lapse<>>>(vars);
```

Comment:
I am not sure what the time derivative should be here, since it will depend on the gauge source function.